openstef/feature_engineering/missing_values_transformer.py

```diff
@@ -69,9 +75,14 @@ def fit(self, x, y=None):
         # Remove always null columns
         is_column_null = x.isnull().all(axis="index")
         self.non_null_feature_names = list(x.columns[~is_column_null])
+        x = x[self.non_null_feature_names]
+
+        # Remove rows with trailing null values
+        self.non_trailing_null_rows = ~x.bfill().isnull().any(axis="columns")
```

Comment:
I believe there are some feature that are null and on which forward imputation should be "okay".
Specifically, I don't think APX feature is ignored. Meaning that when it's used it's always nan for future values.

We should probably use a whitelist for a few features on which forward filling is not okay. For example radiation, windspeed.

![image](https://github.com/user-attachments/assets/a3ae4edd-939e-4804-925e-ec4ab962997d)

Reply:
I added an option to the `MissingValueTransformer`: `no_fill_future_values_features` that defines this whitelist, agree on this setup?